### PR TITLE
sqlpatch crashes due to missing notifier declaration

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: admin/sqlpatch.php [hunk 1]
```diff
@@ -84,6 +84,7 @@
 // NOTE: this line break is intentional!!!!
 
  function executeSql($lines, $database, $table_prefix = '') {
+   global $zco_notifier; 
    @set_time_limit(1200);
 
    global $db, $debug, $messageStack;
```
Comment (drbyte) [inline admin/sqlpatch.php]: Please combine the `globals` on one line.

File: admin/sqlpatch.php [hunk 2]
```diff
@@ -282,7 +283,6 @@ function executeSql($lines, $database, $table_prefix = '') {
           default:
             break;
           } //end switch
-//        } // endif $table_prefix
```
Comment (drbyte) [inline admin/sqlpatch.php]: To remove this requires removing the `if` which opens up at line `114`.

Comment (scottcwilson) [inline admin/sqlpatch.php]: It's commented out, so it's dead code.  It screws up bracket matching to have things like that.

File: admin/sqlpatch.php (hunk 1 above)
Comment (scottcwilson) [inline admin/sqlpatch.php]: fixed

File: admin/sqlpatch.php (hunk 2 above)
Comment (drbyte) [inline admin/sqlpatch.php]: Your bracket-matching comment is understood. But you've removed only the 2nd half of the bracket ... so you're actually creating a bracket-matching problem by not removing the opening bracket up at 113/114.

**Comment by drbyte** [comment]
👍

**Comment by ajeh** [comment]
:+1:

**Pull request merged**